.gitlab-ci.yml

```diff
@@ -292,19 +292,21 @@ build-malus:
   script:
     - git clone
         --depth=1
-        "--branch=$PIPELINE_SCRIPTS_TAG"
+        "--branch=bkchr-update-cargo"
         https://github.com/paritytech/pipeline-scripts
     - ./pipeline-scripts/check_dependent_project.sh
         paritytech
         polkadot
         --polkadot
         "$DEPENDENT_REPO"
         "$GITHUB_PR_TOKEN"
+        $CARGO_UPDATE_CRATES
```

Comment:
```suggestion
        "$CARGO_UPDATE_CRATES"
```
let's be consistent

Reply:
This breaks the script.

Reply:
ah that's a space-separated list, better `{}` then